src/tutorials/ui_reference.md

```diff
@@ -87,7 +87,7 @@ Adjustable properties of the Text Box:
 
 Different UI Elements will have slightly different settings in the properties window, and all of these can be manipulated both in-editor and in code.
 
-!!! info "For Scripting Help see [Lua API](core_api.md) for functions & properties of UI controls."
+!!! info "For Scripting Help see [Lua API](../api/index.md) for functions & properties of UI controls."
```

Comment:
instead of linking to the index, should we link to the separate UI pages?

Reply:
Lets link to UI Controller, since it's easy to miss and contains the most UI functionality.